tests/compas/geometry/test_frame.py

```diff
@@ -68,3 +68,20 @@ def test_frame_predefined():
     assert frame.point == Point(0, 0, 0)
     assert frame.xaxis == Vector(0, 0, 1)
     assert frame.yaxis == Vector(1, 0, 0)
+
```

Comment:
perhaps some other "well-known" interpolation values could be added? e.g. `t=0.25`, `t=0.5`, `t=0.75`